modules/oe_content_consultation/tests/src/Kernel/ConsultationNodeWrapperTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_consultation\Kernel;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\oe_content\EntityWrapperInterface;
+use Drupal\oe_content_consultation\ConsultationNodeWrapper;
+use Drupal\oe_content\CallEntityWrapperInterface;
+
+/**
+ * Tests "Consultation" wrapper class.
+ *
+ * @coversDefaultClass \Drupal\oe_content_consultation\ConsultationNodeWrapper
+ * @group oe_content_consultation
+ */
+class ConsultationNodeWrapperTest extends ConsultationKernelTestBase {
```

Comment:
There is code duplication in ConsultationNodeWrapperTest, CallForProposalsNodeWrapperTest, EventNodeWrapperTest, CallForTendersNodeWrapperTest. Might be trait for node wrappers with methods `testUnsupportedEntityType()`, `testUnsupportedEntityBundle()`, `testGetters()`, `assertGetters` should be created?
@nagyad

Reply:
Why not. Let's open a followup ticket for this.